hyperspy/tests/io/test_mrcz.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2016 The HyperSpy developers
+#
+# This file is part of  HyperSpy.
+#
+#  HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+#  HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
+
+import numpy as np
+import numpy.testing as npt
+import os, os.path, sys
+import tempfile
+import pytest
+from time import perf_counter, sleep
+
+from hyperspy.io import load, save
+from hyperspy.signal import BaseSignal
+from hyperspy._signals.signal2d import Signal2D
+
+#==============================================================================
+# MRCZ Test
+# 
+# Internal python-only test. Build a random image and save and re-load it.
+#==============================================================================
+tmpDir = tempfile.gettempdir()
+mrcName = os.path.join( tmpDir, "testMage.mrcz" )
+MAX_ASYNC_TIME = 2.0
+
+class PythonMrczTests:
```

Comment:
This test class is not discovered by pytest, the name needs to start with "Test".